src/arrays/ArrayUtil.java

```diff
@@ -0,0 +1,168 @@
+package arrays;
+import arrays.ArrayUtilTest;
+
+import java.util.Arrays;
+
+/**
+ * An utility class containing methods for convenient work with arrays
+ */
+public class ArrayUtil {
+
+
+    /**
+     * Create an array with the given size, fill it with the different values and print it.
+     */
+    public static void createArray(int size) {
+        int [] array = {12,45,78,95,23,14,71,65,74};
```

Comment:
Why do we need this array here? 
just iterate over the newly created array and fill with different values, these can be random numbers or just indexes